warehouse/query-core/src/main/java/datawave/query/composite/CompositeTerm.java

```diff
@@ -85,7 +89,17 @@ public void getNodesAndExpressions(List<Class<? extends JexlNode>> nodeClasses,
      * @return
      */
     private String getAppendedExpressions() {
-        return String.join(separator, expressionList);
+        final List<String> newExpressions;
+        if (Iterables.getLast(jexlNodeList) instanceof ASTERNode && jexlNodeList.stream().filter(x -> x instanceof ASTERNode).count() == 1) {
```

Comment:
If I were reviewing this I think I might ask this to be broken into a separate PR; however, it's a relatively small change so I will allow reviewers to comment and dictate whether this is moved elsewhere or can be combined within this PR. This originates from my test branch so this and the matches range function were within the same branch.

Reply:
Broken out into https://github.com/NationalSecurityAgency/datawave/pull/1234 , so will prune this change set from this PR today.